netstandard/ref/System.Diagnostics.CodeAnalysis.cs

```diff
@@ -4,12 +4,12 @@
 
 namespace System.Diagnostics.CodeAnalysis
 {
-    [System.AttributeUsageAttribute((System.AttributeTargets)(748), Inherited=false, AllowMultiple=false)]
+    [System.AttributeUsageAttribute((System.AttributeTargets)(749), Inherited = false, AllowMultiple = false)]
```

Comment:
For those that cannot convert the flag values in your head this just adds assembly to the list of targets :) 

https://github.com/dotnet/corefx/pull/25414/files#diff-bf7a9cbb735f70fb8aaae7ebf2302976R7